src/plugins/discover/public/application/context/context_app.tsx

```diff
@@ -199,6 +199,9 @@ export const ContextApp = ({ indexPattern, anchorId }: ContextAppProps) => {
                 anchorStatus={fetchedState.anchorStatus.value}
                 predecessorsStatus={fetchedState.predecessorsStatus.value}
                 successorsStatus={fetchedState.successorsStatus.value}
+                onEditRuntimeField={() => {
+                  fetchAllRows();
+                }}
```

Comment:
```suggestion
                onEditRuntimeField={fetchAllRows}
```
Can we pass down the same function?

Reply:
So, that did not work I guess?

Reply:
Oops nope it works, I was sure I did this but I guess I forgot to change it.. Fixed now, thanks for catching!